tests/js/spec/views/organizationGroupDetails/groupEventEntries.spec.tsx

```diff
@@ -25,22 +23,11 @@ async function renderComponent(event: Event, errors?: Array<Error>) {
     </OrganizationContext.Provider>
   );
 
-  await tick();
-  wrapper.update();
-
-  const eventErrors = wrapper.find('Errors');
-
-  const alert = eventErrors.find('StyledAlert');
-  const alertSummaryInfo = alert.find('Message');
-
-  alert.simulate('click');
+  const alertSummaryInfo = await screen.findByTestId('event-error-alert');
+  userEvent.click(alertSummaryInfo);
+  const errorItems = await screen.findAllByTestId('event-error-item');
```

Comment:
isn't there any other text we can use to do this check instead of an event id?

Reply:
the tests all expect an array of the event items 🤷 it was using `wrapper.find('ErrorItem');` this is pretty much the same thing